ovirt-engine-keycloak.spec.in

```diff
@@ -32,7 +35,9 @@ Source:		%{name}-@RPM_VERSION@.tar.gz
 
 BuildRequires:	unzip
 
-Requires:	%{name}-setup >= %{version}
+Requires:	%{name}-setup
+Requires:	mod_auth_openidc
```

Comment:
Do we support all possible versions?

Reply:
No idea to be honest - for last 2 years when I started working with keycloak I always installed whatever was available in Centos and it worked. In original blog post introducing ovirt support for keycloak there is no info about version. Do you think it makes sense to do use  >= (whaever is now in centos 8) or leave it unbound ?

Reply:
I decided to add '>=' condition